### PR TITLE
Add a choice for DenseBlock

### LINKED ISSUE #1420: Specify hyperparameter ranges for blocks
### Feature Description
We want to enable the users to specify the value ranges for any argument in the blocks.
The following code example shows a typical use case.
The users can specify the number of units in a DenseBlock to be either 10 or 20.

### Code Example

```python
import autokeras as ak
from kerastuner.engine.hyperparameters import Choice

input_node = ak.ImageInput()
output_node = ak.DenseBlock(num_units=Choice("num_units", [10, 20]))(input_node)
output_node = ak.ClassificationHead()(output_node)
model = ak.AutoModel(input_node, output_node)
```

### Note

Each pull request should only change one hyperparameter in one of the blocks.

### Solution

Example pull requests are #1419 #1425 .
Here are the steps to follow:
1. You can just change any other argument in any other block supported by AutoKeras, as shown [here](https://github.com/keras-team/autokeras/blob/master/autokeras/__init__.py#L16-L38).
2. Change the docstring. [example](https://github.com/keras-team/autokeras/pull/1419/files#diff-7b757ddf51e45c0e02ad8de148d02fd2abd59d670f91d35cac71157e6684c13cL60-R61)
3. Make sure you imported the module. `from kerastuner.engine import hyperparameters`.
4. Change the typing of the argument. [example](https://github.com/keras-team/autokeras/pull/1419/files#diff-7b757ddf51e45c0e02ad8de148d02fd2abd59d670f91d35cac71157e6684c13cL73-R74)
5. Change the saving mechanism to serialized objects. [example](https://github.com/keras-team/autokeras/pull/1419/files#diff-7b757ddf51e45c0e02ad8de148d02fd2abd59d670f91d35cac71157e6684c13cL95-R100)
6. Change the loading mechanism to deserialized objects. [example](https://github.com/keras-team/autokeras/pull/1419/files#diff-7b757ddf51e45c0e02ad8de148d02fd2abd59d670f91d35cac71157e6684c13cR110)
7. Change how we initialize the hyperparameter to self. [example](https://github.com/keras-team/autokeras/pull/1419/files#diff-7b757ddf51e45c0e02ad8de148d02fd2abd59d670f91d35cac71157e6684c13cL80-R85) Copy from where it is originally defined. [example](https://github.com/keras-team/autokeras/pull/1419/files#diff-7b757ddf51e45c0e02ad8de148d02fd2abd59d670f91d35cac71157e6684c13cL115)
8. Change how we use it. [example](https://github.com/keras-team/autokeras/pull/1419/files#diff-7b757ddf51e45c0e02ad8de148d02fd2abd59d670f91d35cac71157e6684c13cL124-R129)

### PR DESCRIPTION
**Pull request by lc0**
### Which issue(s) does this Pull Request fix?

It does not resolve #1420 rather adds a first small piece

### Details of the Pull Request

Just a first PR to see if it makes sense. So I am open for the feedback

### CODE REVIEW

File: autokeras/blocks/basic.py [hunk 1]
```diff
@@ -91,7 +92,13 @@ def __init__(
             int,
         )
         self.use_batchnorm = use_batchnorm
-        self.dropout = dropout
+        self.dropout = utils.get_hyperparameter(
+            dropout,
+            hyperparameters.Choice(
+                "dropout", [0.0, 0.2, 0.3, 0.4, 0.5, 0.6], default=0.0
```
Comment (haifeng-jin) [inline autokeras/blocks/basic.py:98]: We prefer to keep the original search space of [0.0, 0.25, 0.5].
More value would add burden to the search algorithm.

**Review by haifeng-jin — CHANGES_REQUESTED** [review]
Thanks for the PR! It should be good to merge after change.

File: autokeras/blocks/basic.py (hunk 1 above)
Comment (lc0) [reply autokeras/blocks/basic.py]: I reverted back to original search space

**Review by haifeng-jin — APPROVED** [review]
LGTM

**Pull request merged**